### DISCUSSION TITLE
Enable Websocket Connections for Binance

### DISCUSSION THREAD

**Discussion by mertdogan** [Drakkar-Software/OctoBot#1690] [Q&A] [answered]
I manually set "web-socket":true in config.json file and restarted but still error occures as below:

Binance exchange is overloaded by 91%. Binance capacity is 45 simultaneous traded pair/time frame couples and the current configuration involves 86.

What is the way to enable web-socket connections for Binance?

**Comment by Herklos** [comment, marked as answer]
Hi @mertdogan, there is no available websocket for now in OctoBot. We are working on it

**Reply by Herklos** [reply]
Websocket on Binance is available in recent versions.
